test/WebpackOptionsDefaulter.unittest.js

```diff
@@ -155,7 +155,7 @@ describe("WebpackOptionsDefaulter", () => {
 		          "idHint": "vendors",
 		          "priority": -10,
 		          "reuseExistingChunk": true,
-		          "test": /\\[\\\\\\\\/\\]node_modules\\[\\\\\\\\/\\]/i,
+		          "test": /\\[\\\\\\\\\\\\/\\]node_modules\\[\\\\\\\\\\\\/\\]/i,
```

Comment:
Also got problems with this snapshot..

Reply:
But it fails on ci now. This was correct before.

Reply:
Looks like there is a cross platform issue..